### ISSUE TITLE
Generic type aliases don't work

### ISSUE THREAD

**Issue by JelleZijlstra** [python/mypy#2467]
The following produces an error:

```
$ cat generic_alias.py 
from typing import TypeVar, Tuple
S = TypeVar('S')
TInt = Tuple[int, S]
$ python3 -m mypy generic_alias.py 
generic_alias.py:3: error: Invalid type "generic_alias.S"
```

This is part of an example from the documentation at http://mypy.readthedocs.io/en/latest/kinds_of_types.html#type-aliases (third code block).

**Comment by JukkaL** [comment]
This only works on github master right now (there should be a new release in a few days). The docs track the development version.

**Comment by JelleZijlstra** [comment]
Thanks! To reduce future confusion, would it be helpful to mention in the documentation when a feature was introduced, similar to notes in CPython's documentation like "new in version 3.5"?

**Comment by JukkaL** [comment]
That could be well worth it. Another option would be to have the docs point to the latest release by default.

**Comment by gvanrossum** [comment]
FWIW, the convention on readthedocs is that the "stable" version documents
the most recently tagged version (i.e. the latest release) while the
"latest" version documents what was most recently pushed to the repo.

**Comment by JukkaL** [comment]
Closing the issue as generic type aliases are included in the 0.4.6 release. Also created #2476 for linking to the docs for the latest release.
